config.xml

```diff
@@ -11,36 +11,46 @@
         <package name="7zip-15-05.vm"/>
         <package name="apimonitor.vm"/>
         <package name="apktool.vm"/>
+        <package name="bytecodeviewer.vm"/>
         <package name="capa.vm"/>
         <package name="cmder.vm"/>
+        <package name="common.vm"/>
```

Comment:
maybe not needed? I worked off your prior config which seems to have changed now

Reply:
it is not needed, it is a dependency of all packages